### ISSUE TITLE
Proxying with same URL as baseurl  for all users

### ISSUE THREAD

**Issue by rahuldave** [jupyterhub/jupyter-server-proxy#345]
I am trying to run the weights and biases server, which runs under docker, and it seems to expose a problem i am having making jupyterhub the interface into a cluster of gpu machines. Basically, the backend weights and biases server probably needs a baseurl setting to work properly, fince the jupyter-server-proxy rewrites the URL. But there is only one instance of this server I run, and if I am passing the server a baseurl, I cant pass it a different one for each user.

In other words, for a server shared by multiple users but needing a incoming path configuration, what do I do. I couldnt find any docs, so if this is possible and someone can show me the way, I'd love to put in a documentation pull request ...

**Comment by consideRatio** [comment]
This is a bit vague for me to think clearly about as I'm not understanding your situation and the needs you have of this project in your situation. I think though that _if you are using jupyter-server-proxy to start some process_ and you want to inform that process of a base url, you can maybe (depending on the process you start) providing configuration ahead of time or by using a command that includes `{base_url}` that will expand to something of relevance.

Related documentation is here: https://jupyter-server-proxy.readthedocs.io/en/latest/server-process.html#command
